e2e/features/ftpa-respondent-leadership-judge-decision-and-reasons.feature

```diff
@@ -0,0 +1,450 @@
+    And I save my initial appeal
+    And I submit my appeal
+    And I switch to be a `Case Officer`
+    And I request respondent evidence
+    And I upload respondent evidence
+    And I switch to be a `Legal Rep`
+    And I build my case
+    And I submit my case
+    And I switch to be a `Case Officer`
+    And I request respondent review
+    And I add the appeal response
+    And I request appellant review
+    And I request hearing requirements
+    And I switch to be a `Legal Rep`
+    And I submit hearing requirements with all yes
+    And I switch to be a `Case Officer`
+    And I record agreed hearing requirements yes path
+    And I switch to be a `Admin Officer`
+    And I list the case
+    And I switch to be a `Case Officer`
+    And I create case summary
+    And I generate the hearing bundle
+    And I start decision and reasons
+    And I prepare decision and reasons
+    And I send decision and reasons
+    And I switch to be a `Home Office POU`
+    And I apply for ftpa permission to appeal
+
+  @ftpa-respondent-leadership-judge-decision-granted @ftpa-respondent-leadership-judge-decision  @RIA-1434
+  Scenario: FTPA appellant leadership judge decision - Granted
+    When I switch to be a Judge
+    And I select the `Leadership judge FTPA decision` Next step
+    And I am on the `Leadership judge FTPA decision` page
+    And I select `Home Office` for the `Who made the application?` field
+
+    Then I click the `Continue` button
+    And I select `Permission granted` for the `The outcome of the application` field
+    And I add an item to the `FTPA Decision and Reasons document` collection
+    And within the `FTPA Decision and Reasons document` collection's first item, I upload `{@FTPADecisionAndReasons.pdf}` for the `Document` field
+    And within the `FTPA Decision and Reasons document` collection's first item, I type `This is the ftpa decision and reasons` for the `Describe the document` field
+
+    When I click the `Continue` button
+    And I should see the text `Notes for the Upper Tribunal`
+    And I should see the text `Do not issue any case specific directions to the parties regarding the onward conduct`
+    And I should see the text `of the appeal in the Upper Tribunal. The Upper Tribunal will prepare and issue`
+    And I should see the text `standard directions in every case. You should refer the file to the Principal Resident`
+    And I should see the text `Judge of UTIAC if you believe further directions should be issued in advance of the`
+    And I should see the text `initial Upper Tribunal Hearing.`
+    And I click the `There is a point of special difficulty or importance` label
+    And I type `This is information to the upper tribunal` for the `Provide any information that may be helpful to the Upper Tribunal judge (Optional)` field
+
+    When I click the `Continue` button
+    Then I am on the `Check your answers` page
+    And I should see `Respondent` in the `Who made the application?` field
```

Comment:
This step fails as the applicant value is 'Home Office' as in Line 39 and not 'Respondent'. Could you please change it?